### PR TITLE
2 fast 2 furious

### PR DESCRIPTION
**Pull request by deanott**

### CODE REVIEW

**Review by sauyon — CHANGES_REQUESTED** [review]
Please don't remove order by

**Review by sauyon — DISMISSED** [review]

File: pipeline/db/db.go [hunk 1]
```diff
@@ -664,15 +665,13 @@ func GetAltApps(appID string) ([]AltApp, error) {
 	return result, nil
 }
 
-// precentifyArray produces a Postgres compatable 'like any' string intended
-// to be used as part of a larger query.
-func percentifyArray(arr *[]string) {
+func anyPatternFormat(arr *[]string) {
```
Comment (AdamSlack) [inline pipeline/db/db.go]: Equally bad if not worse name

File: pipeline/db/db.go [hunk 2]
```diff
@@ -664,15 +665,13 @@ func GetAltApps(appID string) ([]AltApp, error) {
...
 	for i, v := range *arr {
 		(*arr)[i] = "%" + v + "%"
 	}
 }
 
-func percentifyStartifyArrayify(arr *[]string) {
+func startsWithPatternFormat(arr *[]string) {
```
Comment (AdamSlack) [inline pipeline/db/db.go]: same as above. doesn't indicate what the function does. this name implies that it's checking something and returns a bool

File: pipeline/db/db.go [hunk 3]
```diff
@@ -682,67 +681,170 @@ var appStoreTable = map[string]string{
...
+	*numParam++
+}
+
+// QueryAll depricates all of dean's queries.
+func QueryAll(
 	onlyAnalyzed bool, appStore string, limit string, offset string, developers []string,
 	genres []string, permissions []string, appIDs []string, titles []string, startsWith []string,
 ) ([]AppVersion, error) {
 
+	var querystr string
```
Comment (AdamSlack) [inline pipeline/db/db.go]: Camel case?

**Review by AdamSlack — CHANGES_REQUESTED** [review]

File: pipeline/db/db.go [hunk 4]
```diff
@@ -665,13 +665,13 @@ func GetAltApps(appID string) ([]AltApp, error) {
 	return result, nil
 }
 
-func anyPatternFormat(arr *[]string) {
+func arrRegexAnyFormat(arr *[]string) {
```
Comment (AdamSlack) [inline pipeline/db/db.go]: this is essentially the same. implies that this function is checking if something fits in a specific format

File: pipeline/db/db.go [hunk 5]
```diff
@@ -665,13 +665,13 @@ func GetAltApps(appID string) ([]AltApp, error) {
...
 	for i, v := range *arr {
 		(*arr)[i] = "%" + v + "%"
 	}
 }
 
-func startsWithPatternFormat(arr *[]string) {
+func startsWithRegexFormat(arr *[]string) {
```
Comment (AdamSlack) [inline pipeline/db/db.go]: this is essentially the same. implies that this function is checking if something fits in a specific format

**Review by AdamSlack — CHANGES_REQUESTED** [review]

**Review by AdamSlack — DISMISSED** [review]

**Review by AdamSlack — APPROVED** [review]

**Pull request merged**